tests/component-tests/apiGateway-tests/testCases/UpdateLetterStatus.ts

```diff
@@ -0,0 +1,93 @@
+
+import { RequestHeaders } from '../../../constants/request_headers';
+import { supplierId } from '../../../constants/api_constants';
+
+export type PatchMessageRequestBody = {
+  data: {
+    type: string;
+    id: string;
+    attributes: {
+      reasonCode?: string | number;
+      reasonText?: string;
+      status: string;
+    };
+  };
+};
+
+export type ErrorLink = {
+  about: string;
+};
+
+type PatchErrorResponse = {
+  id: string;
+  code: string;
+  links: ErrorLink;
+  status: string;
+  title: string;
+  detail: string;
+};
+
+export type PatchErrorMessageBody = {
+  errors: PatchErrorResponse[];
+};
```

Comment:
An error response like this will be common to all endpoints, so when we add more test cases probably could refactor this to a common place